### PR TITLE
Average Confidence score utility added

### PR DESCRIPTION
**Pull request by mg6955**
Added utility for calculating average confidence score in utilities/metrics.py
Added steps to test utility function in root directory README.md

### CODE REVIEW

File: utilities/metrics.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+# get git repo root level
+root_path = subprocess.run(['git', 'rev-parse', '--show-toplevel'],
+                           capture_output=True, text=True, check=False).stdout.rstrip('\n')
+#add git repo path to use all libraries
+sys.path.append(root_path)
+from downstream_model import pipeline
+
+def calculateAP(video,model):
+    """
+    Calculates average precision of a video by calculating average precision of each frame
```
Comment (tamannak8) [inline utilities/metrics.py:13]: Change definition from precision to confidence and please mention if it is with respect to a model.

File: utilities/metrics.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    video : string
+    The input video
+    model : model with weights
+    The desired downstream model with appropriate weights
+    Returns
+    ---------
+    video_average_confidence: float
+    The calculated average precision of the video 
+    """
+    video_average_confidence = 0
```
[the remark this answers is not in the document]
Comment (mg6955) [reply utilities/metrics.py]: Done

File: utilities/metrics.py [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+import subprocess
+import sys
+# get git repo root level
+root_path = subprocess.run(['git', 'rev-parse', '--show-toplevel'],
+                           capture_output=True, text=True, check=False).stdout.rstrip('\n')
+#add git repo path to use all libraries
+sys.path.append(root_path)
+from downstream_model import pipeline
+
+def calculateAP(video,model):
```
[the remark this answers is not in the document]
Comment (mg6955) [reply utilities/metrics.py]: Done

File: downstream_model/yolo/visualize.py [hunk 4]
```diff
@@ -60,5 +62,7 @@ def visualize_yolo(image,detection_result) -> np.ndarray:
         cv2.putText(image, result_text, text_location, cv2.FONT_HERSHEY_PLAIN,
                     FONT_SIZE, TEXT_COLOR, FONT_THICKNESS)
         bounding_box_data.append([start_point[0], start_point[1], end_point[0], end_point[1], probability, category_name ])
```
[the remark this answers is not in the document]
Comment (mg6955) [reply downstream_model/yolo/visualize.py]: Done

**Review by tamannak8 — APPROVED** [review]
Approved: everything works

**Pull request merged**